kedro/ipython/__init__.py

```diff
@@ -281,8 +286,12 @@ def _create_cell_with_text(text: str, is_jupyter: bool = True) -> None:
 
 def _print_cells(cells: list[str]) -> None:
     for cell in cells:
-        Console().print("")
-        Console().print(Syntax(cell, "python", theme="monokai", line_numbers=False))
+        if importlib.util.find_spec("rich") is not None:
+            Console().print("")
+            Console().print(Syntax(cell, "python", theme="monokai", line_numbers=False))
```

Comment:
```suggestion
        if rich_console is not None:
            rich_console.Console().print("")
            rich_console.Console().print(rich_syntax.Syntax(cell, "python", theme="monokai", line_numbers=False))
```

Also, just wondering why this doesn't use `get_console` instead of constructing `Console()` twice, or at least use the same console object. 🤔